### PR TITLE
Bug 1206309: Add Edge and Blackberry icons, display tweaks, slug updates

### PR DESCRIPTION
**Pull request by stephaniehobson**
Kumascript template will need to be updated too. New version is on staging: https://developer.allizom.org/en-US/docs/Template%3AEmbedCompatTable

### CODE REVIEW

File: kuma/static/stylus/base/fonts/compat-icons.styl [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    src:url($path-to-fonts + 'compat-icons.eot?-ehmwve');
-    src:url($path-to-fonts + 'compat-icons.eot?#iefix-ehmwve') format('embedded-opentype'),
-        url($path-to-fonts + 'compat-icons.woff?-ehmwve') format('woff'),
-        url($path-to-fonts + 'compat-icons.ttf?-ehmwve') format('truetype'),
-        url($path-to-fonts + 'compat-icons.svg?-ehmwve#compat-icons') format('svg');
+    src:url($path-to-assets + 'fonts/compat-icons.eot?-ehmwve');
+    src:url($path-to-assets + 'fonts/compat-icons.eot?#iefix-ehmwve') format('embedded-opentype'),
+        url($path-to-assets + 'fonts/compat-icons.woff2?-ehmwve') format('woff2'),
+        url($path-to-assets + 'fonts/compat-icons.woff?-ehmwve') format('woff'),
+        url($path-to-assets + 'fonts/compat-icons.ttf?-ehmwve') format('truetype');
```
Comment (openjck) [inline kuma/static/stylus/base/fonts/compat-icons.styl]: Looks like the `$path-to-fonts` change got lost during a rebase.

**Comment by openjck** [comment]
Just a couple of other rebase artifacts:

* We can remove the *media/fonts~HEAD* file
* We should keep 644 permissions for the compat fonts
    1. `cd kuma/static/fonts`
    2. `chmod 644 *`

File: kuma/static/stylus/base/fonts/compat-icons.styl [hunk 2]
```diff
@@ -1,10 +1,10 @@
 @font-face {
     font-family: 'compat-icons';
-    src:url($path-to-fonts + 'compat-icons.eot?-ehmwve');
-    src:url($path-to-fonts + 'compat-icons.eot?#iefix-ehmwve') format('embedded-opentype'),
-        url($path-to-fonts + 'compat-icons.woff?-ehmwve') format('woff'),
-        url($path-to-fonts + 'compat-icons.ttf?-ehmwve') format('truetype'),
-        url($path-to-fonts + 'compat-icons.svg?-ehmwve#compat-icons') format('svg');
```
Comment (openjck) [inline kuma/static/stylus/base/fonts/compat-icons.styl]: Given that the SVG source is removed, can we also remove the *kuma/static/fonts/compat-icons.svg* file?

**Comment by openjck** [comment]
Just a couple of other rebase artifacts:

* We can remove the *media/fonts~HEAD* file
* We should keep 644 permissions for the compat fonts
    1. `cd kuma/static/fonts`
    2. `chmod 644 *`

**Comment by darkwing** [comment]
@stephaniehobson A few updates and I can look at this again.  Or push to `mozilla` repo and I can add commits to make @openjck 's updates.

**Comment by stephaniehobson** [comment]
I think I've dealt with the weird rebase artifacts now.

**Comment by darkwing** [comment]
Looks good _except_ the spot where the "Desktop" icon should show up is blank.  The icon class is `icon-non`.  When I set that to `icon-desktop`, the icon displays as it should.  So we need to update our CSS file or the macro.  If it's the macro you'd like to update, we can R+ and merge this.

**Comment by jwhitlock** [comment]
The icon class should be fixed when https://github.com/mdn/browsercompat/pull/68 is merged

**Comment by darkwing** [comment]
OK, thanks for letting me know!

**Comment by stephaniehobson** [comment]
Yeah, it's not desktop right now, there's a bug in the API where they're all being lumped under "non-browser environments" I need an icon for that but I'm not really sure what to use yet.

**Comment by stephaniehobson** [comment]
No changes, just kicked Travis. I'd merge on @darkwing's R+ above except that he added the "not ready" label a day ago. maybe because Travis needed a kick? Anyway, DW, updates are happening in the macro, I think this is ready.

**Comment by darkwing** [comment]
Oh, sorry @stephaniehobson , I added "not ready" because John mentioned that this PR (https://github.com/mdn/browsercompat/pull/69) was required first.  I see that it's merged so I'll take a look now.

**Comment by darkwing** [comment]
R+!

**Pull request merged**